src/app/domains/wallets/domains.wallets.types.ts

```diff
@@ -6,3 +6,13 @@ export interface IWalletApiResponse {
   creationDate: TServerDateTime;
   name: string;
 }
+
+export interface IWalletModalData {
```

Comment:
Interfaces used on specific place should be in the directory related to that page. In `/domains` you should have types for specific domain. One of the domains we will have will be Wallets. I know this can be difficult to understand so feel free to ask me some questions on priv.
Basically think about domains like "things" we can talk about as user / product owner etc. And pages are a way to display those "things" to user in many different ways.